integrationtests/selector/integrationtest_selector.py

```diff
@@ -1,24 +1,21 @@
-import json
-
 import grpc
-from integrationtests.utils import get_modyn_config
+from integrationtests.utils import get_minimal_pipeline_config, get_modyn_config, init_metadata_db, register_pipeline
 from modyn.selector.internal.grpc.generated.selector_pb2 import (
     DataInformRequest,
     GetAvailableLabelsRequest,
     GetNumberOfPartitionsRequest,
     GetSamplesRequest,
-    JsonString,
-    ModelStoragePolicyInfo,
-    RegisterPipelineRequest,
     SamplesResponse,
-    StrategyConfig,
 )
 from modyn.selector.internal.grpc.generated.selector_pb2_grpc import SelectorStub
 from modyn.utils import grpc_connection_established
 
 # TODO(54): Write more integration tests for different strategies.
 
 
+init_metadata_db(get_modyn_config())
```

Comment:
let's put that in the main body of the test, i.e., in __name__ == "__main__

Reply:
done.